### PR TITLE
Add SSH keys to Vagrant environment

### PR DESCRIPTION
**Pull request by mbarnes**
Allows Ansible to talk to the cluster nodes in the Vagrant environment, so now the host creation (a.k.a. `@slow`) tests that sit and wait for bootstrapping to finish actually work.

(Reviewer should verify that with `behave -D use-vagrant`)

This should also allow us to write tests to actually exercise cluster upgrade, reboot, etc. and catch failed stunts like my threadpool attempt awhile back.

### CODE REVIEW

**Comment by mbarnes** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;UIZ76vxZ&#x2F;399-2-multi-host-mgr-install-ssh-keys-in-vagrant-cluster-nodes"> (2) [multi-host-mgr] Install SSH keys in Vagrant cluster nodes</a>

**Comment by ashcrow** [comment]
Reviewing now

File: features/id_rsa [hunk 1]
```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```
Comment (ashcrow) [inline features/id_rsa]: I wouldn't be a good person if I didn't double check that this is indeed a dummy key. :smile:

Please confirm.

**Comment by ashcrow** [comment]
@mbarnes The actual vagrant work is working great and I'm good with merging.

The change does point out an issue in one of the steps though. I see failures for ```we have a host at {host}``` in [this block](https://github.com/projectatomic/commissaire/blob/7c078d5ccf3387534054fa988a894884aa87c44a/features/steps/hosts.py#L61-L68).

It also is showing a non fatal error though I'm not sure where this is being triggered:
```
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 804, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 757, in run
    self.__target(*self.__args, **self.__kwargs)
  File "[...]/commissaire/src/commissaire/cherrypy_plugins/investigator.py", line 69, in __response_thread
    host, exception = response
TypeError: 'object' object is not iterable
```

Since these issues are technical unrelated to this PR should make issues for these items?

File: features/id_rsa (hunk 1 above)
Comment (mbarnes) [inline features/id_rsa]: Yeah it's a dummy key.

**Comment by mbarnes** [comment]
That might be an uncaught exception in the investigator process.

Unrelated to this PR, but I'll investigate further.

**Comment by ashcrow** [comment]
@mbarnes ok I'll hold off from making either item github issues. Let me know if you decide to split it out or have a fix for it.

**Comment by mbarnes** [comment]
> The change does point out an issue in one of the steps though. I see failures for we have a host at {host} in this block.

I think I overlooked this part, or thought it was the same issue.  What does the failure look like for you?  Is it causing the test to fail?

**Comment by ashcrow** [comment]
@mbarnes  Correct. It causes it to fail. It gets a ``404`` instead of a ``200``.

**Comment by ashcrow** [comment]
@mbarnes looks like the test failure was on me. I didn't re-provision the node.

**Comment by ashcrow** [comment]
LGTM!

**Pull request merged**